src/programming-assistants.bib

```diff
@@ -0,0 +1,68 @@
+@inproceedings{gvero2015synthesizing,
+  title={Synthesizing Java expressions from free-form queries},
```

Comment:
need to put proper nouns inside `{}` so that they show up correctly (e.g. `{Java}`)